site/formulas/decoding.md

```diff
@@ -1 +1,39 @@
 # Decoding to Extended Coordinates
+
+Given 32 octets of data, `s_bytes`, representing a compressed
+Ristretto point, the implementation must:
+
+1. Check that `s_bytes` is in fact 32 octets.  (In typed languages the
+   implementor should guarantee this through the type system, otherwise
+   there should be an explicit check.)
+2. Decode `s_bytes` into a field element, `s`, then re-encode `s` into
+   another array of 32 octets, `s_bytes_check`.  Check that the two
+   arrays `s_bytes` and `s_bytes_check` are equal in constant time.
+   This is done to ensure that the field element `s` satisfies
+   \\( s < p : p \in (2^{255}, 2^{255}-19] \\) (for the case of
+   Ristretto255).  The implementor must
+   ensure that the field element decoding routine ignores the high
+   bit.
```

Comment:
I think we want the opposite: the high bit should be rejected, not ignored?

Reply:
Ah yeah that's what I meant by "ignored" but I see how that could be wrongly interpreted as "we don't care whether it's set or not". Fixed in 9c06283